src/Scope.ts

```diff
@@ -431,16 +442,32 @@ export class Scope {
     }
 
     /**
-     * Find function declarations with the same name as a stdlib function
+     * Find various function collisions
      */
     private diagnosticDetectFunctionCollisions(file: BscFile) {
+        const classMap = this.getClassMap();
         for (let func of file.callables) {
-            if (globalCallableMap.has(func.getName(ParseMode.BrighterScript).toLowerCase())) {
-                this.diagnostics.push({
-                    ...DiagnosticMessages.scopeFunctionShadowedByBuiltInFunction(),
-                    range: func.nameRange,
-                    file: file
-                });
+            const funcName = func.getName(ParseMode.BrighterScript);
+            const lowerFuncName = funcName?.toLowerCase();
+            if (lowerFuncName) {
+
+                //find function declarations with the same name as a stdlib function
+                if (globalCallableMap.has(func.getName(ParseMode.BrighterScript))) {
```

Comment:
Use `lowerFuncName`?

Reply:
oh yeah, good catch! I fixed it and added [this](https://github.com/rokucommunity/brighterscript/pull/246/commits/8d1cdef7c0971cabe9ad3c0c655952768c957772#diff-c9178fef239b00c0b884835914c57f03a67aae8456cad2579a2531ea141d509eL300) unit test to ensure it doesn't regress.